lib/exe/css.php

```diff
@@ -344,6 +360,37 @@ function css_fixreplacementurls($replacements, $location) {
 }
 
 /**
+ * Wrapper for the files, content and mediatype for the event CSS_STYLES_INCLUDED
+ *
+ * @param string $mediatype type ofthe current media files/content set
+ * @param array $files set of files that define the current mediatype
+ * @return array
+ */
+function css_filewrapper($mediatype, $files=array()){
+    return array(
+            'files'                 => $files,
+	        'mediatype'             => $mediatype,
+	        'encapsulate'           => in_array($mediatype, array('screen', 'print')),
```

Comment:
Media types are an important concept of CSS. You cannot simply remove that concept completely and misuse it for other purposes.
I guess your main problem is that we're only using key-value pairs and there's no room for another parameter due to using `.ini` files. Ini files can have arrays, maybe that can be the answer? E.g.
```
css/_admin.css            = screen
css/_admin.css[foo]     = all
```